lib/boilerPlate.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const util = require('util');
+
+/**
+ * Error for whatever you want!
+ *
+ * @extends Error
+ * @class Errors.CoolStuff
+ * */
+class BoilerPlate extends Error{
+    /**
+ * @example
+
+        put your example here
+
+
+ * @example
+ *
+ * ```sh
+ *
+        and the output here
+
+ * ```
+ *
+ *
+ */
+    setMessage(param1, param2){
+        this.name = 'New Error';
+        this.message = `Say something about ${util.inspect(param1, {depth: null, colors: true})} here and something about ${util.inspect(param2, {depth: null, colors: true})}.
+        There`;
+        return this.message;
+    }
```

Comment:
Depth of null is not needed. The standard for all of the error classes should be depth of 5

There is no boilerplate for the documentation of the params, all errors should take an arbitrary fromVar or scope at the end.